components/ChatWindow.tsx

```diff
@@ -220,19 +243,22 @@ export function ChatWindow(props: {
     <>
       <div className="p-4 md:p-8 rounded bg-[#25252d] w-full max-h-[85%] overflow-hidden flex flex-col">
         <h1 className="text-3xl md:text-4xl mb-2 ml-auto mr-auto">
-          {emoji} Fully {browserOnly ? "In-Browser" : "Client-Side"} Chat Over Documents {emoji}
+          {emoji} {titleText} {emoji}
         </h1>
-        <div className="my-4 rounded border p-2 ml-auto mr-auto">
-          <label htmlFor="one">
-            <input id="one" type="checkbox" checked={browserOnly} onChange={(e) => setBrowserOnly(e.target.checked)}/>
-            Browser-only mode
-          </label>
+        <div className="my-4  ml-auto mr-auto">
+          <label htmlFor="modelSelector" className='mr-2'>Model provider</label>
+          <select id="modelSelector" value={modelProvider} onChange={e => setModelProvider(e.target.value as ModelProvider)} className="p-2 bg-white text-black rounded border">
```

Comment:
I would actually love to keep Ollama the default here since it's a lot more effective with the 7B parameter models

And would like to keep it as a toggle too - am thinking toggle + select once `Browser Only` is selected. But I can make that change later!

Reply:
set ollama as default but kept select for simplicity